src/main/java/org/galatea/starter/entrypoint/RestExceptionHandler.java

```diff
@@ -44,6 +46,16 @@
     return buildResponseEntity(error);
   }
 
+  @ExceptionHandler(ConstraintViolationException.class)
+  protected ResponseEntity<Object> handleConstraintViolation(
+      final ConstraintViolationException exception) {
+    log.debug("Invalid input data sent", exception);
+
+    String errorMessage = "Invalid input data.  Please consult the documentation.";
```

Comment:
Can we give the caller more info about what data failed validation? Like when filling a web form, would be useful to know what field fell foul of the rules.